Transformer/Tests/ConditionFromBooleanToMongoTransformerTest.php

```diff
@@ -1,93 +0,0 @@
-<?php
```

Comment:
@itkg-ngilain As we do not use anymore this class, test won't be usefull for us. This was discussed during code coverage meeting. Test for deprecated class were always removed at this point. But we can change the method if everybody is ok, @alavieille , @itkg-ngilain , @betd-rcarles , @djamnazi